frontend/containers/dashboard/account-information/basic-info/component.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react';
+
+import { FormattedMessage, FormattedDate, useIntl } from 'react-intl';
+
+import { translatedLanguageNameForLocale } from 'helpers/intl';
+
+import Loading from 'components/loading';
+
+import { useAccount } from 'services/account';
+
+import { AccountBasicInfoProps } from './types';
+
+export const AccountBasicInfo: FC<AccountBasicInfoProps> = ({}: AccountBasicInfoProps) => {
+  const intl = useIntl();
+
+  const { userAccount, userAccountLoading } = useAccount({ includes: ['owner'] });
+
+  return (
+    <div className="relative p-6 bg-white rounded-lg">
+      <div className="flex text-sm">
+        <div className="flex-grow font-semibold">
+          <FormattedMessage defaultMessage="Basic info" id="Py+eVV" />
+        </div>
+      </div>
+      {userAccountLoading ? (
+        <div className="flex items-center justify-center w-full h-10">
+          <Loading visible={true} iconClassName="w-10 h-10" />
+        </div>
+      ) : (
+        <>
+          <div className="text-sm gap-6 grid grid-cols-1 lg:grid-cols-[240px_1fr] mt-8 text-gray-900">
+            <span className="text-gray-600">
+              <FormattedMessage defaultMessage="Date of creation" id="KxPY7j" />
+            </span>
+            <span>
+              {!!userAccount?.created_at && FormattedDate({ value: userAccount.created_at })}
+            </span>
+
+            <span className="text-gray-600">
+              <FormattedMessage defaultMessage="Language" id="y1Z3or" />
+            </span>
+            <span>{translatedLanguageNameForLocale(intl, userAccount?.language)}</span>
+            <span className="text-gray-600">
+              <FormattedMessage defaultMessage="Owner" id="zINlao" />
+            </span>
+            <span>
+              {userAccount?.owner.first_name} {userAccount?.owner.last_name}
+            </span>
+          </div>
+          <div className="flex flex-col items-center gap-4 p-4 mt-6 text-sm md:flex-row rounded-xl bg-background-middle">
+            <div className="flex flex-grow">
+              <FormattedMessage
+                defaultMessage="To update any of this information, or to delete the account, please contact the platform administrator."
+                id="/XKclL"
+              />
```

Comment:
Shouldn't we update this sentence now that the owner can delete the account? Maybe something like:
>To update any of this information, please contact the platform administrator. To delete the account, please contact the account owner.